info/v1/container.go

```diff
@@ -842,6 +842,22 @@ type ProcessStats struct {
 	Ulimits []UlimitSpec `json:"ulimits,omitempty"`
 }
 
+type MemoryBandwidthMonitoringStats struct {
+	// The 'mbm_total_bytes'.
+	TotalBytes uint64 `json:"mbm_total_bytes,omitempty"`
+
+	// The 'mbm_local_bytes'.
+	LocalBytes uint64 `json:"mbm_local_bytes,omitempty"`
+
+	// The 'llc_occupancy'.
+	LLCOccupancy uint64 `json:"llc_occupancy,omitempty"`
```

Comment:
It is not memory bandwidth related at all :P

Reply:
Yes, but it's part of Memory Bandwidth Monitoring. I decided to add metrics for all MBM features.

Reply:
Last level cache occupancy comes from Cache Monitoring Technology rather than from Memory Bandwidth Monitoring. See: https://01.org/cache-monitoring-technology

Reply:
Ok, I see now. I thought that it's part of MBM because of ``/sys/fs/resctrl/info/L3_MON/mon_features``.